content/rest/authentication.textile

```diff
@@ -29,7 +29,7 @@ jump_to:
     - Related Types#related-types
 ---
 
-Ably clients can authenticate with Ably using "Basic Authentication":/general/authentication#basic-authentication (which uses the customer's "API key":https://support.ably.io/solution/articles/3000030054-what-is-an-app-api-key), "Token Authentication":/general/authentication#token-authentication (which relies on a token customers obtain from Ably), or a token request that customers sign and issue to their clients. "Token Authentication":/general/authentication#token-authentication, in most cases, is the recommended strategy as it provides robust access control and stringent security measures.
+Ably clients can authenticate with Ably in one of two ways. They can use "Basic Authentication":/general/authentication#basic-authentication or "Token Authentication":/general/authentication#token-authentication. Basic Authentication makes use of the customer's "API key":https://support.ably.io/solution/articles/3000030054-what-is-an-app-api-key) to connect with Ably. Token Authentication requires a server to provide an "Ably Token":/general/authentication#tokens, an "Ably TokenRequest":#request-token, an "Ably JWT (JSON Web Token)":/general/authentication#ably-jwt, or an "External JWT":https://jwt.io with an embedded Ably--compatible token to the client. "Token Authentication":/general/authentication#token-authentication, in most cases, is the recommended strategy due to it providing robust access control and stringent security measures.
```

Comment:
I've not gone through this in detail but obviously some of the changes in the general doc need to be reflected here.